intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/breadcrumbs/YangBreadcrumbsProvider.java

```diff
@@ -42,6 +42,10 @@ public boolean acceptElement(@NotNull PsiElement psiElement) {
     @NotNull
     @Override
     public String getElementInfo(@NotNull PsiElement psiElement) {
-        return Objects.requireNonNull(PsiTreeUtil.findChildOfType(psiElement, YangIdentifierArgImpl.class)).getText();
+        try {
+            return Objects.requireNonNull(PsiTreeUtil.findChildOfType(psiElement, YangIdentifierArgImpl.class)).getText();
```

Comment:
Objects.requireNonNull will produce NPE as it is designed to do so... Much better fix would be to check what is actually null, in this case it is probably PsiTreeUtil.findChildOfType(psiElement, YangIdentifierArgImpl.class) and so requireNonNull will throw NPE...

so if you extract PsiTreeUtil.findChildOfType(psiElement, YangIdentifierArgImpl.class) into variable and then do something like `return var != null ? var.getText() : "";` you will be much better off in both readability, and it will make much more sense... Also you can get rid of Objects.requireNonNull as this can clearly be null...